src/components/ProgramForm/Context/index.tsx

```diff
@@ -370,6 +371,7 @@ export const AppContext: React.FC<{ children: React.ReactNode; id: number }> = (
                                   ...data.data,
                                   isNew: false,
                                 };
+                                history.push(`/courses/list/${id}/program/?topic=${newTopic.id}`);
```

Comment:
Can I change the url to a new topic from here? I was not able to get the response from `updateTopic` to have the id to a given topic after saving

Reply:
`data.data.id` should have `id` of created topic

Reply:
@qunabu ok done